src/wallet/wallet2.cpp

```diff
@@ -510,7 +519,7 @@ std::pair<std::unique_ptr<wallet2>, password_container> wallet2::make_from_file(
 std::pair<std::unique_ptr<wallet2>, password_container> wallet2::make_new(const boost::program_options::variables_map& vm)
 {
   const options opts{};
-  auto pwd = get_password(vm, opts, true);
+  auto pwd = get_password(vm, opts, true, true);
```

Comment:
Why is this one forcing true ? wallet2 is used both by CLI and RPC wallets. Is it possible to make it a parameter like above ?

Reply:
Sounds good -- it definitely should be a parameter since the RPC logic for creating new wallets uses it. I made a bad assumption there.